src/cloud-api-adaptor/pkg/securecomms/ppssh/ppssh.go

```diff
@@ -0,0 +1,304 @@
+package ppssh
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sync"
+
+	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/pkg/securecomms/sshproxy"
+	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/pkg/securecomms/sshutil"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	PROVEN_PP_PRIVATE_KEY_PATH  = "/tmp/provenPpPrivateKey"
+	PROVEN_WN_PUBLIC_KEY_PATH   = "/tmp/provenTePublicKey"
```

Comment:
Curious why the const name contains `WN` but the file name contains `Te` (and frankly, I have no idea what each of these stands for).

Reply:
Tnx for catching this. 
WN stand for Worker Node
PP STand for Peer Pod
WN and PP are the two peers of the added ssh channel.